src/DDTrace/Integrations/Integration.php

```diff
@@ -33,6 +33,14 @@ public function __construct()
         $this->configuration = $this->buildConfiguration();
     }
 
+    public function addTraceAnalyticsIfEnabledLegacy(Span $span)
+    {
+        if (!$this->configuration->isTraceAnalyticsEnabled()) {
+            return;
+        }
+        $span->setMetric(Tag::ANALYTICS_KEY, $this->configuration->getTraceAnalyticsSampleRate());
+    }
+
```

Comment:
Is this related to these changes or just something you grouped in here while passing by?

Reply:
This [answer](https://github.com/DataDog/dd-trace-php/pull/877/files#r424546314) applies to this comment too.